feature/onboard/src/main/java/team/ppac/onboard/login/mvi/LoginSideEffect.kt

```diff
@@ -2,4 +2,6 @@ package team.ppac.onboard.login.mvi
 
 import team.ppac.common.android.base.UiSideEffect
 
-sealed class LoginSideEffect: UiSideEffect
\ No newline at end of file
+sealed class LoginSideEffect: UiSideEffect
+
+data object NavigateToSample : LoginSideEffect()
```

Comment:
```suggestion
sealed class LoginSideEffect: UiSideEffect {
    data object NavigateToSample : LoginSideEffect()
}
```
로 하는 편인듯? 아님 sealed interface를 사용하더라구용?

Reply:
오오.. 맞네유 수정하겠슴다~